examples/goto.py

```diff
@@ -28,7 +28,10 @@ async def run():
 
     await asyncio.sleep(1)
 
-    await drone.action.goto_location(55.8688660, -4.2851267, 40, 0)
+    await drone.action.goto_location(47.399386, 8.535245, 500, 0)
```

Comment:
Change heading angle to `float('nan')` such that `await drone.action.goto_location(47.399386, 8.535245, 500,float('nan')`

Reply:
Oh yes, that would be nicer.